pkg/reconciler/apiserversource/controller_test.go

```diff
@@ -20,6 +20,8 @@ import (
 	"os"
 	"testing"
 
+	filteredFactory "knative.dev/pkg/client/injection/kube/informers/factory/filtered"
+
```

Comment:
As `SetupFakeContext()` uses the fake informers, you also need to import the fake informers (like in line 38), to register these informer factories:
```
	_ "knative.dev/pkg/client/injection/kube/informers/factory/filtered/fake"
	_ "knative.dev/pkg/client/injection/kube/informers/rbac/v1/role/filtered/fake"
	_ "knative.dev/pkg/client/injection/kube/informers/rbac/v1/rolebinding/filtered/fake"
```

Welcome to faking :D